src/kernel_config_manager.sh

```diff
@@ -131,15 +130,14 @@ function save_config_file()
     printf '%s\n' "$description" > "$metadata_dir/$name"
   fi
 
+  if cmp -s "$original_path/.config" "$dot_configs_dir/$configs_dir/$name"; then
```

Comment:
For string concatenation, we use:
https://kworkflow.org/content/codingstyle.html#string-concatenation